qcodes/instrument_drivers/Galil/dmc_41x3.py

```diff
@@ -0,0 +1,964 @@
+"""
+This file holds the QCoDeS driver for the Galil DMC-41x3 motor controllers.
+
+Colloquially known as the "stepper motors".
+"""
+from typing import Any, Dict, Optional, List, Tuple
+import numpy as np
+
+from qcodes.instrument.base import Instrument
+from qcodes.instrument.channel import InstrumentChannel
+from qcodes.utils.validators import Enum, Ints, Multiples
+
+try:
+    import gclib
+except ImportError as e:
+    raise ImportError(
+        "Cannot find gclib library. Download gclib installer from "
+        "https://www.galil.com/sw/pub/all/rn/gclib.html and install Galil "
+        "motion controller software for your OS. Afterwards go "
+        "to https://www.galil.com/sw/pub/all/doc/gclib/html/python.html and "
+        "follow instruction to be able to import gclib package in your "
+        "environment."
+    ) from e
+
+
+class GalilMotionController(Instrument):
+    """
+    Base class for Galil Motion Controller drivers
+    """
+
+    def __init__(self, name: str, address: str, **kwargs: Any) -> None:
+        """
+        Initializes and opens the connection to the Galil Motion Controller
+
+        Args:
+            name: name for the instrument
+            address: address of the controller
+        """
+        super().__init__(name=name, **kwargs)
+        self.g = gclib.py()
+        self.address = address
+        self.open()
+
+    def open(self) -> None:
+        """
+        Open connection to Galil motion controller. This method assumes that
+        the initial mapping of Galil motion controller's hardware's mapping
+        to an IP address is done using GDK and the IP address in burned in.
```

Comment:
"is"? and "burned in" - is that how they call it? :)
```suggestion
        to an IP address is done using GDK and the IP address is burned in.
```

Reply:
yes :)